### DISCUSSION TITLE
Error Invalid read of size 4

### DISCUSSION THREAD

**Discussion by adribastiasm** [IIC2133-PUC/2024-1#56] [Tarea 0]
Hola! Estoy corriendo los testcases y todo el rato me sale el siguiente error:
Invalid read of size 4
==99==    at 0x48B4A94: __vfscanf_internal (vfscanf-internal.c:345)
==99==    by 0x48B423C: __isoc99_fscanf (isoc99_fscanf.c:30)
==99==    by 0x10951E: main (main.c:37)
==99==  Address 0xc0 is not stack'd, malloc'd or (recently) free'd

y no logro entender ni encontrar el error, porque la linea 37 del main.c es:
  result = fscanf(input_file, "%d", &N_PLANETAS);
Eso es código que nos dieron uds. y no lo he cambiado.
Me podrían ayudar a entender y solucionar el error por favor.

**Comment by benjavicente** [comment]
Hola! Por favor, sube lo que llevas, e indica como estas corriendo el test. Si es error en código que te entregamos, probablemente estes corriendo mal, pasandole un archivo de input que no existe.

**Reply by PabloNavarroY** [reply]
(Subirlo al repositorio, no acá)

**Reply by adribastiasm** [reply]
ahí lo subi.
Lo estoy corriendo con docker
cd edd-docker
make start
cd T0-2024-1-adribastiasm
make
 valgrind ./ship ./tests/parte-1-1/input.txt output.txt

**Reply by adribastiasm** [reply]
ya lo resolví
